### ISSUE TITLE
Pre-train without label

### ISSUE THREAD

**Issue by anndymaktub** [Omid-Nejati/MedViTV2#7]
Thank you for your excellent paper. Do you think the results would be better if the model was pretrained using SSL methods beforehand?

**Comment by Omid-Nejati** [comment]
Thank you for your interest in our study! Pretraining the model using SSL (self-supervised learning) methods could indeed potentially lead to improved results. However, I haven't personally checked this yet.
If you need anything for your exploration, please feel free to contact me!
